tools/wt_verify/wt_verify.py

```diff
@@ -109,6 +109,9 @@ def parse_metadata(line):
         line = line[3:-1]
         for x in line.split(" | "):
             [key, value] = x.split(": ", 1)
+            if len(value) == 0:
+                dict[key] = value
```

Comment:
Do we want to save anything at all? It looks like the dict is saving (str,int), saving an empty string does not seem correct.